legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/protocols/pure/v1_20_0/models/metamodel_connection.pure

```diff
@@ -33,6 +33,11 @@ Class meta::protocols::pure::v1_20_0::metamodel::store::relational::connection::
 {
 }
 
+Class meta::protocols::pure::v1_20_0::metamodel::store::relational::connection::alloy::authentication::DeltaLakeAuthenticationStrategy extends meta::protocols::pure::v1_20_0::metamodel::store::relational::connection::alloy::authentication::AuthenticationStrategy
```

Comment:
See comments in the legend-engine PR for modeling this as a vault reference etc

Reply:
Created an authentication strategy generic enough for `APIToken` - see latest commit